src/features/typeDecorations.ts

```diff
@@ -21,26 +45,27 @@ export default () => {
             !vscode.languages.match(enableLanguages, textEditor.document)
         )
             return
-        const { selections } = textEditor
+        const { selections, document } = textEditor
         const pos = selections[0]!.end
-        const { document } = textEditor
         const text = document.lineAt(pos).text.slice(0, pos.character)
         const match = /(:| =) $/.exec(text)
         textEditor.setDecorations(decoration, [])
         if (!match) return
         const offset = match[0]!.length
         const hoverData: vscode.Hover[] = await vscode.commands.executeCommand('vscode.executeHoverProvider', document.uri, pos.translate(0, -offset))
         let typeString: string | undefined
+
+        const isInStyles = await checkIfInStyles(document, pos)
         for (const hover of hoverData) {
             const hoverString = hover.contents
                 .map(content => {
                     if (typeof content === 'object') return content.value
                     return content
                 })
                 .join('')
-            const typeMatch = /: (.+)/.exec(hoverString)
+            const typeMatch = isInStyles ? /Syntax: (.*)/.exec(hoverString) : /: (.+)/.exec(hoverString)
             if (!typeMatch) continue
-            typeString = typeMatch[1]!
+            typeString = unescape(typeMatch[1]!).replaceAll('\\', '')
```

Comment:
Added this to remove annoying `\` in output 
![image](https://user-images.githubusercontent.com/74474615/176426355-9ff4c1fd-ef2b-452a-a973-65f6fa2e3705.png)